web-app/HeimScripts/heatmap/run.R

```diff
@@ -104,6 +106,30 @@ buildFields <- function(df){
   return(df)
 }
 
+writeDataForZip <- function(df, patientIDs) {
+  pidCols    <- as.character(patientIDs)
+  t          <- df
+  t[pidCols] <- lapply(t[pidCols], function(v) { (v - df$MEAN) / df$SD })
```

Comment:
Becaus ethe other zscore is calculated on unpivoted data. I agree that it kind of sucks we're calculating the zscore twice, but I guess that it's faster this way than pivoting again the unpivoted data calculated in `buildFields`.